<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindow</class>
 <widget class="QMainWindow" name="ProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профиль пользователя</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="statusTip">
   <string>Личный кабинет пользователя</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistory">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>История игр</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Приветствуем тебя, </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>user.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Побед:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Bal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImage">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить аватарку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваш ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
